timescaledb/tutorials/grafana/visualizations/pie-chart.md

```diff
@@ -0,0 +1,161 @@
+
+1.  In the Grafana dashboard, in the `Dashboard variable` field, select the stocks
+    to graph. Adjust the time range of the dashboard if needed. Make sure the
+    returned data has a column named `time` that contains timestamps. The
+    timestamps should be in ascending order. Otherwise, you get an error. The
+    returned data looks like this:
+
+    <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/piechart/tabledata1.png" alt="Screenshot of the table view of valid time-series data for stocks."/>
+
+1.  In the `Visualizations` field, select `Pie chart`. Grafana turns the query
+    into a pie chart. This example shows a pie chart price distribution of JPM, IBM, AAPL, AMD, and CVS stocks which has 22%, 23%, 24%, 15%, and 16%, respectively, within a specific period. The returned data had lots of data, but only the lowest values for each stock were displayed because it was the selected option in the calculation field.
+
+    <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/piechart/pie+chart.png" alt="Screenshot of the pie chart produced by Grafana. The pie chart represents the price and percentage of Google in the past 3 months."/>
+
+1.  How data is displayed depends on how the value options are refined on the dashboard.
+    There are two  options for choosing how much information is displayed: Calculate, which reduces each value to a single series, and All values, which shows every value from a single series.
+    Displaying all the values from the returned data, select ‘all value’ in the value option, show specific fields to display and the limit of the returned data:
+
+    <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/piechart/datadisplaytype.png" alt="Screenshot of the all values shown in pie chart produced by Grafana. The pie chart represents the price of selected stocks in the past 3 months."/>
+
+1.  The new chart looks like this, with a limit of 40:
+
+   <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/piechart/piechart2.png" alt="Screenshot of the all values shown in pie chart produced by Grafana. The pie chart represents the price of selected stocks in the past 3 months."/>
+
+</procedure>
+
+## Create a donut chart with volume transactions
+
+A  donut chart is just a pie chart with the middle cut out. People favor them because the human eye often finds it easier to differentiate when looking at arcs not slices. A donut can help you compare respective categories or dimensions to the larger whole. Each donut arc has the same width but a different length. So when you compare which one is greater, you only have to make one point of comparison—which has a longer length?
```

Comment:
```suggestion
A donut chart is a pie chart with the middle cut out. Compared to a pie chart,
it can be easier to compare segments on a donut chart because the human eye
is better at comparing the lengths of arcs. Otherwise, they're exactly the same.
```